pkg/kubectl/cmd/cmd.go

```diff
@@ -363,8 +362,12 @@ func (h *DefaultPluginHandler) Lookup(filename string) (string, bool) {
 }
 
 // Execute implements PluginHandler
-func (h *DefaultPluginHandler) Execute(executablePath string, cmdArgs, environment []string) error {
-	return syscall.Exec(executablePath, cmdArgs, environment)
```

Comment:
The main reason we originally went with `syscall.Exec` is that under the covers this calls `execve` which replaces the old process with the new one (see http://man7.org/linux/man-pages/man2/execve.2.html). Can you verify if something like https://stackoverflow.com/a/23365445/606583 works on windows, I'd prefer this approach conditionally on windows than entirely replacing the current approach.

Reply:
SGTM. exec.Command under the hood uses StartProcess and does the necessary setup.
For window now we use exec.Command and other platforms uses syscall.Exec. PTAL.
Thanks for the review.

Reply:
I'm actually not sure if we can do exec.Command. These are fundamentally different things.

`exec.Command#Start` calls `os.StartProcess` and `syscall.Exec` directly makes the system call. Both have very different code paths, especially on different platforms like Windows vs POSIX systems.

If we are making this change we should at least make sure
- inherited env vars work
- inherited std streams (stdin stdout stderr) work
- exit code of plugin determines exit code of the kubectl invocation
- there's no parent kubectl process (manually verify this!)

Reply:
Yes exec.Command is different.

As mentioned there is no support for exec syscall on windows hence the failure. Updated changes now uses exec.Command on windows and keeps behavior same for other platforms.  exec.Command takes cares of std streams, environment variables.  I have verified the behavior on windows.